coderd/prometheusmetrics/prometheusmetrics_test.go

```diff
@@ -511,23 +511,27 @@ func TestAgentStats(t *testing.T) {
 func TestExperimentsMetric(t *testing.T) {
 	t.Parallel()
 
+	if len(codersdk.ExperimentsAll) == 0 {
+		t.Skip("No experiments are currently defined; skipping test.")
+	}
```

Comment:
@dannykopping I see these prom metrics are somewhat assuming that the `len(codersdk.ExperimentsAll)` will always be at least > 0, but with the removal of shared ports we currently have an empty slice for it and it's failing these tests. Let me know if you'd like me to do something besides skipping these under that condition in this PR. Thanks

Reply:
We spoke offline, he gave me the 👍 for now and we'll make the changes to get these tests running again soon.